styleguide/source/assets/js/fitvids.js

```diff
@@ -69,7 +69,7 @@
           var videoID = 'fitvid' + Math.floor(Math.random()*999999);
           $this.attr('id', videoID);
         }
-        $this.wrap('<div class="fluid-width-video-wrapper"></div>').parent('.fluid-width-video-wrapper').css('padding-top', (aspectRatio * 100)+"%");
+        $this.wrap('<div class="fluid-width-video-wrapper"></div>').parent('.fluid-width-video-wrapper').css('padding-top', (56)+"%");
```

Comment:
Is this a change to the "core" fitvids.js script that we're including? If it is, it might be more future-friendly to put this in our own JS if possible? Otherwise, I'll also be happy with an explanation as to why it's fine where it is